tests/src/Functional/StatusReportTest.php

```diff
@@ -96,7 +96,7 @@ public function testInvalidCredentials() {
     $key->save();
 
     $this->drupalGet($status_report_path);
-    $this->assertSession()->pageTextContains('Unauthorized');
+    $this->assertSession()->pageTextContains('Invalid host or public key. Please provide the valid host and key value.');
```

Comment:
This change is deprecated by #193.

Reply:
If we would commit this change as-is then after the next MGMT API server release - that hopefully fixes this invalid/incorrect message - test builds would be broken again.

Reply:
At that point we will know it's been fixed/changed. I suggest we leave this as we don't know if or when this change will be reverted.